### DISCUSSION TITLE
Eliminazione metadati dei file

### DISCUSSION THREAD

**Discussion by marcomeneghin** [globaleaks/globaleaks-whistleblowing-software#3691] [Q&A]
La piattaforma permette già quanto in oggetto?

**Comment by evilaliv3** [comment]
È un argomento complesso.

- nessuna normativa obbliga la cancellazione automatica dei metadati
- preservare i metadati di un originale è necessario per fine di prova
- cancellare dati in modo automatico potrebbe portare a data breach per
cancellazioni di informazioni rilevanti

Per questo si ritiene che una corretta implementazione debba eventualmente
permettere di redarre selettivamente e manualmente alcune parti o al più
permettere di scaricare per fine di lavorazione una copia dei dati
edulcorata dai metadati.

L'argomento è trattato nella sezione della documentazione:
https://docs.globaleaks.org/en/main/security/ThreatModel.html#metadata
